internal/testrunner/runners/system/runner.go

```diff
@@ -339,6 +339,11 @@ func (r *runner) createAgentInfo(policy *kibana.Policy, config *testConfig) (age
 	info.Agent.Runtime = config.Agent.Runtime
 	info.Agent.PidMode = config.Agent.PidMode
 
+	// If user is defined in configuration file has preference ?
+	if info.Agent.User == "" && packageManifest.Agent.Privileges.Root {
+		info.Agent.User = "root"
+	}
```

Comment:
As it is, if one configuration defines another user, that user will have preference over what the package manifest sets. 

Should it be set always according to what the package manifest sets ignoring the setting from the configuration file?